infra/main.bicep

```diff
@@ -0,0 +1,117 @@
+targetScope = 'subscription'
+
+@minLength(1)
+@maxLength(64)
+@description('Name of the the environment which is used to generate a short unique hash used in all resources.')
+param environmentName string
+
+@minLength(1)
+@description('Primary location for all resources (filtered on available regions for Azure Open AI Service).')
+@allowed([ 'westeurope', 'southcentralus', 'australiaeast', 'canadaeast', 'eastus', 'eastus2', 'francecentral', 'japaneast', 'northcentralus', 'swedencentral', 'switzerlandnorth', 'uksouth' ])
+param location string
+
+//Leave blank to use default naming conventions
+param resourceGroupName string = ''
+param identityName string = ''
+param apimServiceName string = ''
+param logAnalyticsName string = ''
+param applicationInsightsDashboardName string = ''
+param applicationInsightsName string = ''
+
+param openAiInstances object = {
```

Comment:
I didn't notice any instructions on updating this list, and especially calling out that the number of instances is flexible (I believe). Did I miss it?